src/locationTracker/LocationTracker.js

```diff
@@ -28,16 +29,21 @@ class LocationTracker extends React.Component {
                     <ul style={{ padding: '2%' }}>
                         {
                             this.props.logic.areas().filter((area) => !areaBlacklist.includes(area)).map((value) => (
-                                <div className="group-container" onClick={this[_.camelCase(`open${value}`)]} onKeyDown={this.onClick} role="button" tabIndex="0">
-                                    <h3 style={{ cursor: 'pointer', color: this.props.colorScheme.text }}>
-                                        {value}
-                                        <AreaCounters
-                                            totalChecksLeftInArea={this.props.logic.getTotalCountForArea(value)}
-                                            totalChecksAccessible={this.props.logic.getInLogicCountForArea(value)}
-                                            colorScheme={this.props.colorScheme}
-                                        />
-                                    </h3>
-                                </div>
+                                <Row>
+                                    <div className="group-container" onClick={this[_.camelCase(`open${value}`)]} onKeyDown={this.onClick} role="button" tabIndex="0">
+                                        <h3 style={{ cursor: 'pointer', color: this.props.colorScheme.text }}>
+                                            {value}
+                                            <AreaCounters
+                                                totalChecksLeftInArea={this.props.logic.getTotalCountForArea(value)}
+                                                totalChecksAccessible={this.props.logic.getInLogicCountForArea(value)}
+                                                colorScheme={this.props.colorScheme}
+                                            />
+                                        </h3>
+                                    </div>
+                                    <div>
+                                        <WotHHint />
```

Comment:
Sorry I don't know react but why is `width` a required prop, but not passed in? Either remove the prop from the component or pass it in here